playbooks/engineer-workflow.md

```diff
@@ -79,11 +79,36 @@ Once again, the title should concisely explain the change or addition. The descr
 - Finally, any pre- or post-deploy migration steps should be clearly explained (see
   [migrations](data-migrations.md)).
 
-_Assign_ another team member to the PR. All team members are encouraged to contribute, but that team member has the
-explicit responsibility to carefully review, consider any unanticipated impact, monitor discussion, and ultimately
-merge or close the PR. You may also want to add reviewers to request other developers' specific input.
+**_Request a review_** from at least one team member. This should be someone who has context around the changes
+that were made, experience with the system(s) being modified, and/or knowledge of the language/framework in use.
+Team members that are not explicitly assigned to a PR are encouraged to contribute as well, but reviewers have the
+explicit responsibility to help make sure the code is elegant, understandable, and will not have unforseen impacts.
```

Comment:
I think that we might want to punt on what makes a good code review for a separate PR / RFC.

In particular, I think it's difficult to expect reviewers to anticipate unforeseen impacts because they're by definition unanticipated. We should do our best to point out undesired behavior and vulnerabilities but a reviewer can't anticipate everything that might go wrong. That's why we have practices to roll out changes gradually and guarding new functionality behind feature flags, which minimizes risk.

I also think a requirement to ensure that code is elegant is too ambiguous.

At its simplest, a reviewer has the responsibility to read the code in question and provide commentary on it.

There are plenty of ways we can expand on the expectations, but I think it might be better suited as a separate addition.
- Ensure that the code accomplishes the author’s purpose.
- Ensure that the code is correct, well-architected, secure, and maintainable.
- Ensure that the code is covered by tests
- Etc

https://medium.com/palantir/code-review-best-practices-19e02780015f
https://engineeringblog.yelp.com/2017/11/code-review-guidelines.html

Reply:
that makes sense! happy to amend those lines to make them more specific and actionable, and I think a future RFC would be a good idea as well.